custom/icds/tasks.py

```diff
@@ -106,16 +105,24 @@ def get_language_code(user_id, telugu_user_ids, marathi_user_ids):
 
 
 @periodic_task(
-    run_every=crontab(hour=9, minute=0, day_of_week='mon'),
+    run_every=crontab(hour=9, minute=0),
     queue=settings.CELERY_PERIODIC_QUEUE,
     ignore_result=True
 )
-def run_weekly_indicators(phased_rollout=True):
+def run_user_indicators(phased_rollout=True):
     """
-    Runs the weekly SMS indicators Monday at 9am IST.
-    If it's the first week of the month, also run the monthly indicators.
+    Runs the weekly / monthly user SMS indicators at 9am IST.
+    This task is run every day and the following logic is applied:
+        - if it's Monday, the weekly indicators are sent
+        - if it's the first of the month, the monthly indicators are sent
```

Comment:
Should this be true even if the first of the month is Sunday? For some reason I thought that the gov didn't like doing that kind of thing on sundays, but I could just be misremembering

Reply:
I'll doublecheck with the project team about this point, but the new requirement is to send on the first of the month

Reply:
actually, we're already sending other messages on Sundays. I'll still doublecheck whether we should be sending on Sundays or not but if we need to impose that restriction i think we'd just do so in the SMS settings for the project so that all messages would just queue up on Sundays and then get sent on Monday